ot/bregman.py

```diff
@@ -258,10 +259,6 @@ def sinkhorn2(a, b, M, reg, method='sinkhorn', numItermax=1000,
         return sinkhorn_stabilized(a, b, M, reg, numItermax=numItermax,
                                    stopThr=stopThr, verbose=verbose, log=log,
                                    **kwargs)
-    elif method.lower() == 'sinkhorn_epsilon_scaling':
```

Comment:
Why did you remove this algorithm?

This is not a problem that appeared in #230

Reply:
OK then you can remove it from sinkhorn 2

Reply:
but please update the test and title of the PR so that it reflects everything that the PR does.

Reply:
I am not sure how I can update the test. It was already catching for the ValueError, but it's just that the ValueError effectively received had to do to a logic error and not catching that the method was not supported

Reply:
So I would argue that the test is correct, but the error raised if the method is not supported should be a NotImplementedError

Reply:
Do you want me to change the valuerror to notimplemetederror?

Reply:
OK i meant text in the PR sorry (above), you remove a method in sinhorn2 it should appear in the TEXT of the PR

Reply:
Alrighty

Reply:
What about this ValueError thing_